src/libraries/System.Console/src/System/IO/StreamWriter.iOS.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.IO
+{
+    internal class IOSStreamWriter : StreamWriter
+    {
+        private readonly List<byte> _acc = new List<byte>();
```

Comment:
What does `acc` mean/stand for? I think we should use whole words for the field.